CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing guide for Gregorio
+
+## Reporting an issue
+
+You can report a bug and request a new feature on the [bug tracker](https://github.com/gregorio-project/gregorio/issues).
+
+Please search for existing issues before reporting a new one. 
+
+Please do not use the issue tracker for personal support requests, instead use the mailing-list (https://gna.org/mail/?group=gregorio) or IRC: #gregorio on freenode.
+
+### Report a bug
+
+When you report a bug, please provide the following information:
+
+ * version of Gregorio
+ * a minimal example (gabc or gabc+tex if relevant) showing the bug
+ * a screenshot of the wrong result
+ * a precise description (if possible with a picture) of the expected output
+
+### Request feature
+
+When requesting a feature, please provide the following informations:
+
+ * a minimal example (gabc or gabc+tex) that currently doesn't work, but would
+   if the feature was done
+ * a precise description (if possible with a picture) of the expected output
+
+## Contributing code
+
+### Development environment
+
+It is not currently possible to compile Gregorio under Windows. The build requirements
+are the same under GNU/Linux and Mac OSX:
+
+ * [git](http://git-scm.com/)
+ * the [GNU Build System](http://airs.com/ian/configure/) (often referred to as "Autotools")
+ * [gcc](https://gcc.gnu.org/) (and associated tools)
+ * [GNU indent](https://www.gnu.org/software/indent/)
+
+### Coding style
+
+The code uses two spaces for indentation. The C code follows the [GNU coding standards](https://www.gnu.org/prep/standards/html_node/Writing-C.html).
+
+Use indent on your code before commiting it, with the `.indent.pro` file in repository's root folder.
```

Comment:
Perhaps a little more detail here?  Or at least a link to a page explaining how to do this?  I have to be honest that I'm one of the ones guilty violating this (2 spaces just isn't enough for me to see the indention structure) and need to implement the conversion process to clean up my code.